eo-runtime/src/main/java/EOorg/EOeolang/EOio/EOstdin$EOnext_line.java

```diff
@@ -59,7 +61,17 @@ public class EOstdin$EOnext_line extends PhDefault {
                                 "There is no line in the standard input stream to consume"
                             );
                         }
-                        return new Data.ToPhi(sc.nextLine());
+                        final String line = sc.nextLine();
+                        if (sc.hasNextLine()) {
+                            final StringBuilder leftlines = new StringBuilder();
+                            while (sc.hasNextLine()) {
+                                leftlines.append(sc.nextLine()).append("\n");
```

Comment:
@includealex I think it would be better to replace new line separator to a system-independent line separator, like `System.lineSeparator()`